packages/edit-post/src/components/welcome-guide/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __experimentalGuide as Guide } from '@wordpress/nux';
+import { __ } from '@wordpress/i18n';
+import { __experimentalCreateInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { CanvasImage, EditorImage, BlockLibraryImage, InserterIconImage } from './images';
+
+export default function WelcomeGuide() {
+	const areTipsEnabled = useSelect( ( select ) => select( 'core/nux' ).areTipsEnabled() );
+
+	const { disableTips } = useDispatch( 'core/nux' );
+
+	if ( ! areTipsEnabled ) {
+		return null;
+	}
+
+	return (
+		<Guide
+			className="edit-post-welcome-guide"
+			finishButtonText={ __( 'Get started' ) }
```

Comment:
I find it interesting that when you click the finish button you actually get started 😄 

Did you consider using some alternatives?

Reply:
I think `finishButton` is correct. It's the button that, when clicked, marks the guide as _finished_ and calls `onFinish`. 

In our case, `WelcomeGuide` shows the `Guide` to brand-new users and so _finishing the guide_ is equivalent to _getting started with the editor_. But `Guide` is generic and could be used in other cases e.g. a plugin tutorial that displays a guide when a _Help_ button is clicked.